variables.tf

```diff
@@ -22,10 +22,10 @@ variable "s3_bucket_policy" {
   description = "Additional S3 bucket policy"
 }
 
-variable "gitlab_project_id" {
```

Comment:
Can't we keep`gitlab_project_id` for the sake of backward compatibility and turn it into default value for `gitlab_project_ids`

Reply:
I meant keeping backward compatibility in the main module here, not in gitlab submodule.